server/consumer.go

```diff
@@ -116,8 +116,17 @@ type SequenceInfo struct {
 type CreateConsumerRequest struct {
 	Stream string         `json:"stream_name"`
 	Config ConsumerConfig `json:"config"`
+	Action ConsumerAction `json:"action" `
```

Comment:
Any concern from an API standpoint that the client will need to know what 0, 1, and 2 refer to? Is it worth adding `UnmarshalJSON` to `ConsumerAction` to allow a string and handle decoding to an enum?

Reply:
Yes we should do that like store does for storage type etc.

Reply:
Yes, I'm planning to add that shortly.
That was the initial reason I used const.
Enum is fine, but needs that addition.
Will add it shortly.